plugins/by-name/trouble/default.nix

```diff
@@ -115,200 +115,248 @@ helpers.neovim-plugin.mkNeovimPlugin {
   ];
 
   settingsOptions = {
-    position =
-      helpers.defaultNullOpts.mkEnum
-        [
-          "top"
-          "left"
-          "right"
-          "bottom"
-        ]
-        "bottom"
-        ''
-          Position of the list.
-        '';
-
-    height = helpers.defaultNullOpts.mkInt 10 ''
-      Height of the trouble list when position is top or bottom.
+    auto_close = defaultNullOpts.mkBool false ''
+      Automatically close the list when you have no diagnostics.
     '';
 
-    width = helpers.defaultNullOpts.mkInt 50 ''
-      Width of the list when position is left or right.
+    auto_preview = defaultNullOpts.mkBool true ''
+      Automatically preview the location of the diagnostic.
+      <esc> to close preview and go back to last window.
```

Comment:
I think the `<esc>` may be stripped by nixos-render-docs as a potential html tag.

We probably want `` `<esc>` `` anyway though.